terminal.go

```diff
@@ -464,10 +468,18 @@ func (g *LightningTerminal) start() error {
 	case <-readyChan:
 
 	case err := <-g.errQueue.ChanOut():
-		return err
+		g.statusMgr.SetErrored(
```

Comment:
We also need to set `lnd` as errored on line 444. I tested this in integrated mode where `lnd` couldn't start because of an already occupied listener port. The status shows `running: false` but `error: ""`.

Reply:
And then, when trying to shut down (Ctrl+C), it just logs `LTND: Gracefully shutting down.` but doesn't quit... This could be hard to pin down though, so low prio/non-blocking.

Reply:
for both of these, the issue is pre-existing. 
Currently (master branch) if you already have LND running & then try starting LiT in integrated mode, things hang on the `lnd.Main(...)` line. If you then shutdown (Ctrl+C) it will also just log `LTND: Gracefully shutting down.` and will log `LTND: Shutdown complete` about 10 seconds later but still the `lnd.Main(...)` method wont return and so Litd doesnt exit.  

I think the issue is that things block here: https://github.com/lightninglabs/lightning-terminal/blob/03f4bf99a9fd0313a8f219492dcdf25a88817169/terminal.go#L547

I'll open a separate PR to fix this. 

The one open question is: if the port is occupied, `lnd.Main` does not return. It hangs and retries. So there really isnt any way for LiT to set the `error:` in the status. So I think for now, just showing "running: false" is good enough since that should indicate to the user that something is wrong.